app/javascript/components/partners/campaign_creator/fields/CampaignCreatorPhoneNumber.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useFormikContext } from "formik";
+import { CampaignCreatorField } from "components/partners/campaign_creator/fields/CampaignCreatorField";
+import { CampaignCreatorInput } from "components/partners/campaign_creator/fields/CampaignCreatorInput";
+
+export const CampaignCreatorPhoneNumber = ({
+  vaccinationCenterPhoneNumber,
+}) => {
+  const fieldName = "phoneNumber";
+  const { setFieldValue, values } = useFormikContext();
+  return (
+    <>
+      <CampaignCreatorField
+        label="Numéro de téléphone à communiquer aux volontaires (facultatif)"
```

Comment:
Comme c'est déjà dans la section _Informations pour les
volontaires_, je pense qu'on peut simplifier
```suggestion
        label="Numéro de téléphone (facultatif)"
```